twilight-http/src/request/guild/create_guild/builder.rs

```diff
@@ -107,7 +107,10 @@ impl RoleFieldsBuilder {
 
     /// Set the role color.
     ///
-    /// This must be a valid hexadecimal RGB value.
+    /// This must be a valid hexadecimal RGB value. `0x000000` is ignored
```

Comment:
this documentation can be added more places, such as https://github.com/twilight-rs/twilight/blob/d968c2c83b505d2ccce9c748f44b7aad74593dc8/twilight-http/src/request/guild/role/create_role.rs#L80-L85